mix.exs

```diff
@@ -19,7 +19,7 @@ defmodule Postgrex.Mixfile do
 
   # Configuration for the OTP application
   def application do
-    []
+    [applications: [:logger]]
```

Comment:
No, but formatting error reports is useful in development and testing. Should we set it based on `Mix.env`?